### ISSUE TITLE
AI/human player switch

### ISSUE THREAD

**Issue by wenzhe-li** [tonyxu99/String_fighter_AI#2]
Thanks for sharing this very cool work!

I am interested in training agents on both sides of the game (act on P1 and P2), which requires learning against the game AI on both sides. I think this is very related to your extension on 2P mode, as well as switching between human and AI. Would you mind giving me some advice on activating/deactivating game AI on both sides?

Many thanks in advance!

**Comment by tonyxu99** [comment]
Yes, that's my next goal. I will think about it and update the code.

**Comment by tonyxu99** [comment]
Just added AI vs AI test script which can allow 2 AIs to fight each other. Please see the README.

**Comment by wenzhe-li** [comment]
Thank you for the update! I am also wondering if we can replace model1/model2 with the original level12 AI provided by the emulator, so that we can train one model playing the left side, as well as another model for the right side. I guess this is a little different with the 1P mode, as the previous initialization specifying players=1 always set the player(trainer) on one particular side (for instance, Ryu in RyuVsGuile).

**Comment by tonyxu99** [comment]
> Thank you for the update! I am also wondering if we can replace model1/model2 with the original level12 AI provided by the emulator, so that we can train one model playing the left side, as well as another model for the right side. I guess this is a little different with the 1P mode, as the previous initialization specifying players=1 always set the player(trainer) on one particular side (for instance, Ryu in RyuVsGuile).

Thank you for your feedback! I have updated the project. Please see the README for how to train the Player2 AI. https://github.com/tonyxu99/String_fighter_AI/blob/main/README.md#train-an-ai-to-play-as-player-2-right-side
